Fill.py

```diff
@@ -244,8 +244,8 @@ def distribute_items_restrictive(world, gftower_trash=False, fill_locations=None
             continue
 
         gftower_trash_count = (
-            random.randint(15, 50) if world.goal[player] in {'triforcehunt', 'localtriforcehunt'} else random.randint(0,
-                                                                                                                      15))
+            random.randint(15, 50) if world.goal[player] in {'triforcehunt', 'localtriforcehunt', 'ganontriforcehunt', 'localganontriforcehunt'}
```

Comment:
I think we're getting to the point where we want if 'triforcehunt' in world.goal[player]

Reply:
Resolved in f9c40a8